app/views/layouts/_footer.html.erb

```diff
@@ -0,0 +1,23 @@
+<html>
+<head>
+<meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
Here in Rails we do not need these tags. Please, consider removing <html>, <head>, <meta> and its closing tags